docker/solver/Dockerfile

```diff
@@ -1,6 +1,6 @@
 ARG expose_via=local
 
-FROM golang:latest as base
+FROM golang:latest AS base
```

Comment:
You've made the little ocd person in me very happy

Reply:
> You've made the little ocd person in me very happy

I've never met a warning I didn't want to fix :)